mission-4/app.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import fs from 'fs';
+import usersRouter from "./src/routes/users.router.js";
+import productsRouter from './src/routes/products.router.js';
+import articlesRouter from './src/routes/articles.router.js';
+import commentsRouter from './src/routes/comments.router.js'; 
+
+// uploads 폴더가 존재하지 않으면, 폴더를 생성합니다.
+const dir = './uploads';
+if (!fs.existsSync(dir)) {
+  fs.mkdirSync(dir);
+}
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.json()); //body-parser middleware
+app.use('/api', [usersRouter, productsRouter, articlesRouter, commentsRouter]);
+
+app.use((err, req, res, next) => {
```

Comment:
관습적으로 사용하지 않는 파라미터의 경우에는 `_`(언더바)를 붙이기도 합니다. 필수는 아니고 참고해주세요!
```suggestion
app.use((err, _req, res, _next) => {
```